index.html

```diff
@@ -17,12 +17,17 @@
   <header>  
     <nav class="header" id="logo">
       <h3 class="poppins"><a href="#">My Logo</a></h3>
-      <img src="./assets/hamburger menu.png">
+      <img class="hamburger" src="./assets/hamburger menu.png">
+      <ul class="nav-list">
+        <li>Portfolio</li>
+        <li>About</li>
+        <li>Contact</li>
+      </ul>
     </nav>
   </header>
   
   <section class ="first-wrapper">
-
+    <div class="text-wrapper">
     <h1 class="poppins">I'm Victoria Glad to see you!</h1>
```

Comment:
- [ ] Please use the font size that the desktop template uses. It is normal that on bigger screens font size is higher than in the mobile version.  You can check their values by inspecting the desktop template 😄 

Please note that this is happening in almost every section ☝️

Reply:
Thank you for the feedback. I have made these changes in the current PR.